src/models/user.js

```diff
@@ -17,51 +18,94 @@ const user = (sequelize, DataTypes) => {
 			type: DataTypes.STRING
 		},
 		token: {
-			type: DataTypes.STRING,
-			required: true
-		}
-	},
-	{
-		getterMethods: {
-			getPassword() {
-				return this.password;
-			}
+			type: DataTypes.STRING
+		},
+		email: {
+			type: DataTypes.STRING
+		},
+		phone: {
+			type: DataTypes.STRING
 		}
 	},
 	{
 		schema: process.env.DATABASE_SCHEMA
 	});
 
-	User.findByLogin = async (login,password) => {
-		const user = await User.findOne({
+	/**
+	 * Looks up and validates a user by username and password.
+	 *
+	 * @param {String} login The username or email of the user.
+	 * @param {String} password The password of the user.
+	 *
+	 * @return {String} returns either the valid login token or an error message.
+	 */
+	User.findByLogin = async (login, password) => {
```

Comment:
Maybe think about changing the param from `login` to `username`. Makes it easier to understand what you are actually looking for in the database

Reply:
Added it that way so a user can login via email _or_ username. For strictly-API usage I would agree, but this system will be logging in regular users of a front-end as well. Letting them use their email address might be beneficial. 

We might want to make email input required when we add those fields.

Reply:
I think we might want to change it so that its just the email for login and then we change username to displayName so that users can put in whatever they want to have displayed on the front-end for themselves inside the system.